src/EditorFeatures/Core/GoToDefinition/GoToDefinitionCommandHandler.cs

```diff
@@ -98,8 +99,6 @@ internal static bool TryExecuteCommand(Document document, int caretPosition, IGo
                 var notificationService = workspace.Services.GetService<INotificationService>();
                 notificationService.SendNotification(errorMessage, title: EditorFeaturesResources.Go_to_Definition, severity: NotificationSeverity.Information);
             }
-
-            return true;
         }
```

Comment:
this method always returned 'true', so could just simplify the stack all the way up.